src/Columns/ReverseIndex.h

```diff
@@ -325,9 +325,16 @@ class ReverseIndex
     static constexpr bool use_saved_hash = !is_numeric_column;
 
     UInt64 insert(const StringRef & data);
+
+    /// If index is not built, builds it.
     UInt64 getInsertionPoint(const StringRef & data);
+
+    /// Returns the found index if the #index is built, otherwise, searches for it linearly.
```

Comment:
Not true, it only does a hash-based lookup

Reply:
Yup, forgot to replace the comment.